WalletWasabi/Models/ChaumianCoinJoin/CcjClientState.cs

```diff
@@ -67,6 +67,27 @@ public bool Contains(SmartCoin coin)
 			}
 		}
 
+		public bool Contains(params TxoRef[] txos)
+		{
+			lock (StateLock)
+			{
+				foreach (TxoRef txo in txos)
+				{
+					if (WaitingList.Keys.Any(x => x.GetTxoRef() == txo))
+					{
+						return true;
+					}
+
+					if (Rounds.Any(x => x.CoinsRegistered.Any(y => y.GetTxoRef() == txo)))
```

Comment:
Am I loosing my mind or this method was using linq before?

Reply:
At first I did it like this. Then I created this PR on top of the other PR. Then I updated the other PR with the linq, but not this one. Rebasing to the master now.